docker-compose.full-demo.yml

```
@@ -0,0 +1,158 @@
+version: '3.2'
```

```suggestion
---
version: "3.2"
```
The doc is missing `---` at the start of the `yml` file.

We should also standarize on using single or double quotes in our various Compose files. I would probably advise in favor of double quotes.

---

@alessfg It seems like many of your comments are related to YAML formatting.  Do you have a preferred YAML formatting tool (ideally that runs on the CLI) that I can apply to the whole repo that will do things like prefer double quotes?

My preferences are:
* Runs in JS (since users of this demo will need to have nodejs installed)
* Opinionated (because I don't want to curate a file full of rules)

In this org, I'm formatting all my JS with [prettier](https://prettier.io/) which fits the bill but I'm curious if you're ok with the output.  I formatted this PR as a sample here: https://github.com/microservices-march/platform/pull/4.

Seems like prettier won't fix the issue above so I'll be changing that 👍